controller/space.go

```diff
@@ -19,7 +19,9 @@ import (
 )
 
 const (
-	spaceResourceType = "space"
+	// APIStringTypeCodebase contains the JSON API type for codebases
+	APIStringTypeSpace = "spaces"
+	spaceResourceType  = "space"
 )
 
 var scopes = []string{"read:space", "admin:space"}
```

Comment:
Well, I tried :) But turned out that you can't declare a const array in go. Array is a dynamic thing. But these scopes are more like placeholders for now. We set them but not really using. So, we may change them later.